src/main/java/org/apache/commons/crypto/random/CryptoRandomFactory.java

```diff
@@ -198,14 +198,6 @@ public static CryptoRandom getCryptoRandom(final Properties props)
             } catch (final Exception e) {
                 lastException = e;
                 errorMessage.append("CryptoRandom: [" + className + "] failed with " + e.getMessage());
-            } catch (final ExceptionInInitializerError initializerError) {
```

Comment:
`ExceptionInInitializerError` is uniformly handled in `ReflectionUtils`, and an exception is thrown at the same time. So, there is no need to handle it here.